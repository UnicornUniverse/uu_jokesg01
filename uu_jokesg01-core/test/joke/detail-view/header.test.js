import { mount, wait } from "uu5g05-test";
import Header from "../../../src/joke/detail-view/header.js";

describe(`UuJokesCore.Joke.DetailView.Header`, () => {
  it(`with joke prop`, async () => {
    const joke = {
      name: "Test joke",
      visibility: false,
    };

    const wrapper = mount(<Header joke={joke} />);

    await wait();
    expect(wrapper).toMatchSnapshot();
  });
});
