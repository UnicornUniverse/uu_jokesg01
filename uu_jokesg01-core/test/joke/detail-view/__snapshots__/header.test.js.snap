// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UuJokesCore.Joke.DetailView.Header with joke prop 1`] = `
<UuJokesCore.Joke.DetailView.Header
  joke={
    Object {
      "name": "Test joke",
      "visibility": false,
    }
  }
>
  <Uu5Elements.withTooltip(Uu5Elements.Text)>
    <Uu5Elements.Text>
      <span
        className="uuelements-0 uuelements-0"
      >
        <Uu5Elements.withTooltip(Uu5Elements.Text)
          colorScheme="building"
          significance="subdued"
        >
          <Uu5Elements.Text
            colorScheme="building"
            significance="subdued"
          >
            <span
              className="uuelements-0 uuelements-15k0dco"
            >
              <Uu5Elements.withTooltip(Uu5Elements.Icon)
                className="uujokescore-joke-detailview-12z0wuy"
                icon="mdi-eye-off"
              >
                <Uu5Elements.Icon
                  className="uujokescore-joke-detailview-12z0wuy"
                  icon="mdi-eye-off"
                >
                  <span
                    className="uuelements-4g9xqe mdi mdi-eye-off uujokescore-joke-detailview-12z0wuy"
                  />
                </Uu5Elements.Icon>
              </Uu5Elements.withTooltip(Uu5Elements.Icon)>
            </span>
          </Uu5Elements.Text>
        </Uu5Elements.withTooltip(Uu5Elements.Text)>
        Joke
         - 
        Test joke
      </span>
    </Uu5Elements.Text>
  </Uu5Elements.withTooltip(Uu5Elements.Text)>
</UuJokesCore.Joke.DetailView.Header>
`;
